server_test.go

```diff
@@ -59,7 +59,9 @@ func RunLocalUDPServerWithFinChan(laddr string) (*Server, string, chan struct{},
 	fin := make(chan struct{}, 0)
 
 	go func() {
-		server.ActivateAndServe()
+		if err := server.ActivateAndServe(); err != nil {
+			panic(err)
```

Comment:
hmmm, `panic` in a lib
This was probably to reason to, uhm, ignore that error

Reply:
Forgot to remove this. It was returning an error and stalling all of the tests so I needed to figure out what the error was.